content/influxdb/v1.8/tools/influx_inspect.md

```diff
@@ -452,6 +454,78 @@ The flag to report exact cardinality counts instead of estimates.
 Default value is `false`.
 Note: This can use a lot of memory.
 
+### `report-disk`
+
+Use the `report-disk` command to review disk usage by shards and measurements for TSM files in a specified directory. Useful for determining disk usage for capacity planning and identifying which measurements or shards are using the most space.
+
+Calculates the total disk size in bytes by database (`db`), retention policy (`rp`), shard (`shard`), tsm file (`tsm_file`), and measurement (`measurement`).
+
+#### Syntax
+
+```
+influx_inspect report-disk [ options ]
+```
+
+#### Options
+
+Optional arguments are in brackets.
+
+##### [ `-detailed` ]
+
+Include this flag to report disk usage by measurement.
+
+##### -datadir <data_dir>
```

Comment:
this param doesn't appear to be defined in the code (at least not when I tested it.).  I'm going to file an issue to make sure that it does.  It means that the examples below should also have -datadir in front of the location... ~/.influxdb/data  --- should be:
influx_inspect report-disk -datadir ~/.influxdb/data

Reply:
I believe the way this is documented for other params when the path to the datadir is provided but it is not and explicitly named parameter is using `<path>`.  See also: https://github.com/influxdata/influxdb/issues/21007

Reply:
Thank you, Tim, dropped `-datadir` and added `<path>`
Will stay tuned from @lesam to confirm this implementation.

Reply:
Yes that is correct. See a usage example at https://github.com/influxdata/influxdb/issues/21007#issuecomment-805303519